### PR TITLE
May be good NAN prevention?

### PR DESCRIPTION
**Pull request by Bod9001**
May be good NAN prevention

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs [hunk 1]
```diff
@@ -29,10 +29,10 @@ public class GasMix
 		public float Temperature
 		{
 			get { return temperature; }
-			set { temperature = Math.Clamp(value, 0, Single.MaxValue); }
+			set { temperature = Math.Clamp(value, 0.001f, Single.MaxValue); }
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs:32]: explain

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs:32]: I'm thinking maybe something tries to go 
n / temperature

**Comment by AtnerNT** [comment]
A big issue with the NaN and inf exploits is gases behave very strangely if you have both water and a gas in the same pipe. Water gets in pipes from turbines and I'm assuming pipes dont have a way to handle both liquid and gas so that can be leveraged for the exploits. While this will solve temp / 0s I dont think it will fix the NaN exploits using water.
Might wanna talk to Butch for clarification how the NaN exploit is done

**Comment by Bod9001** [comment]
yeah
I think I did a while ago, forgot though xD

**Pull request closed without merging**